.github/workflows/scheduled.yml

```
@@ -164,7 +165,7 @@ jobs:
           mv -v api-samples _includes/api-samples
 
       - name: Build site
-        run: bundle exec rake build
+        run: /usr/bin/rake build
```

```suggestion
        run: bundle exec rake build
```

looks like this is the only one that needs it.

could even try:

```suggestion
        run: bundle exec jekyll build
```

and further cleanup the `Rakefile`.